Gemfile

```diff
@@ -36,14 +43,18 @@ end
 
 # see http://projects.puppetlabs.com/issues/21698
 platforms :mswin, :mingw do
+  gem "ffi", "~> 1.9", :require => false
   gem "sys-admin", "~> 1.5.6", :require => false
-  gem "win32-dir", "~> 0.3.7", :require => false
-  gem "win32-eventlog", "~> 0.5.3", :require => false
-  gem "win32-process", "~> 0.6.5", :require => false
-  gem "win32-security", "~> 0.1.4", :require => false
-  gem "win32-service", "~> 0.7.2", :require => false
-  gem "win32-taskscheduler", "~> 0.2.2", :require => false
-  gem "win32console", "~> 1.3.2", :require => false
+  gem "win32-api", "~> 1.4.8", :require => false
+  gem "win32-dir", "~> 0.3", :require => false
+  gem "win32-eventlog", "~> 0.5", :require => false
+  gem "win32-process", "~> 0.6", :require => false
+  gem "win32-security", "~> 0.1", :require => false
+  gem "win32-service", "~> 0.7", :require => false
+  gem "win32-taskscheduler", "~> 0.2", :require => false
+  gem "win32console", "~> 1.3", :require => false
+  gem "windows-api", "~> 0.4", :require => false
+  gem "windows-pr", "~> 1.2", :require => false
```

Comment:
Making this less restrictive so that bundler, using the puppet version, can figure out the exact version dependencies here.